packages/minter-contracts/src/michelson-to-ts.ts

```diff
@@ -33,7 +33,7 @@ const outPath = args['out-path'];
 const michelsonPath = args['michelson-path'];
 const endpoint = args['endpoint'];
 const michelsonToJson = (fileName: string) => new Promise<string>((resolve, reject) => {
-  child.exec(`TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER=yes ${tzClientPath} -E ${endpoint} convert script ${fileName} from michelson to JSON`, (err, stdout, stderr) => {
+  child.exec(`TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER=yes ${tzClientPath} -E ${endpoint} convert data ${fileName} from michelson to JSON`, (err, stdout, stderr) => {
     if (stderr) {
       $log.error(stderr);
     }
```

Comment:
`package.json` `test` and `test-contracts` scripts seems to need to call `michelson-to-ts` script as well.
Can you run all the contract tests by simply
* `git clone`
* `yarn install`
* `yarn test`
?

Reply:
I verified that running `yarn michelson-to-ts` generates the exact same typescript for existing contracts after making this change. 

Also regarding tests, `yarn test` still fails for the marketplace tests and for `hello.test.ts`. I think the latter can just be deleted and I will make an issue for the former.

Reply:
1. Right now `yarn test` is working because we cache all compilation-generated files in our repo (which might change in future). Even though, if some developer would try to modify a view and check if tests pass, he would need to remember to run `compile-contracts` and `michelson-to-ts` manually before executing the test. `yarn-test` should be self sufficient and guarantee that all generated files are up-to-date. Currently it calls `compile-contracts` yarn script but not `michelson-to-ts`

2. `hello.test.ts` just checks that it can interact with the sandbox node RPC. Strictly, it is not needed, but I added it earlier just to test that bootstrapping works. You may remove it, but I would investigate why it is failing

3. Some market contract test failed for me as well because they have dependency on `tezos-client`. I never managed to make them work on my machine, but it also worth looking into it and try to eliminate dependency on `tezos-client` (not a part of this PR)

Reply:
1. I agree. However, I think we will actually be able to completely delete `michelson-to-ts` soon by using ligo to directly compile the contracts to json  and not store the `.tz` files. This will also eliminate the dependency on `tezos-client`. See #107. Adding `yarn michelson-to-ts` to the test script in package.json is more annoying than i thought because yarn does not have a good way to capture arguments and pass them to commands in the middle of a script, and it is necessary to pass the path to the `tezos-client` as an argument to `michelson-to-ts`. I think we should just continue cacheing the json in this PR. I will open an MR for deleting michelson-to-ts soon.
2. I just needed to extend the jest timeout and it passes now. 
3. See 1